fivegla_visualization/device_measurement/fivegla_visualization_device_measurement.py

```diff
@@ -76,6 +96,21 @@ def draw(self):
         :return: None
         """
         selected_entity_id = self.dlg.cmbEntityId.currentText()
+
+        # store start- and end-date from dialog box with .date()-method
+        start_date = self.dlg.dateEditStart.date()
+        end_date = self.dlg.dateEditEnd.date()
+
+        if start_date > end_date:
+            MessageBox.show_error_box("No valid date range!")
+            return
+
+        # reformat the stored data from dialog-box in string-format (yyyy-MM-ddTHH:mm:ss.zzzZ)
+        start_date_time = QDateTime(start_date, QTime(0, 0, 0, 0))
+        end_date_time = QDateTime(end_date, QTime(23, 59, 59, 999))
+        start_date_string = start_date_time.toString("yyyy-MM-ddTHH:mm:ss.zzzZ")
+        end_date_string = end_date_time.toString("yyyy-MM-ddTHH:mm:ss.zzzZ")
```

Comment:
Das Verwenden von Typen in Variablennamen sollte nach Clean Code vermieden werden, da es weniger Chancen für das Refactoring bietet.